secure_message/repository/database.py

```diff
@@ -100,11 +95,10 @@ def _populate_events(self, message):
             elif row.event == EventsApi.READ.value:
                 message['read_date'] = str(row.date_time)
 
-    def _populate_to_and_from(self, actor, message):
+    def _populate_to_and_from(self, user, message):
         for row in self.statuses:
-            if row.actor == actor:
+            if row.actor == user.user_uuid or (user.is_internal and (row.actor == constants.NON_SPECIFIC_INTERNAL_USER or row.actor == constants.BRES_USER)):
```

Comment:
Is this here for the purposes of passing tests?

Reply:
No , its needed so that the status's are retrieved correctly in the case that a respondent sends a message to a group . If the user is internal we want to see the status's even if the message was sent to a group .